src/instructlab/configuration.py

```diff
@@ -125,6 +126,10 @@ def TAXONOMY_DIR(self) -> str:
     def CHATLOGS_DIR(self) -> str:
         return path.join(self._data_dir, STORAGE_DIR_NAMES.CHATLOGS)
 
+    @property
+    def E2E_DIR(self) -> str:
+        return path.join(self._data_dir, STORAGE_DIR_NAMES.E2E)
```

Comment:
You'll also need to make sure that this is getting created in the `ensure_storage_directories_exist` function below.

Reply:
ack